include/mapper/mapper_class.h

```diff
@@ -101,6 +103,20 @@ class MapperClass {
   void PublishRadiusMarkers(const Eigen::Vector3d &center,
                             const double &radius);
 
+  // Publish no-fly-zones and planning zone for Rviz visualization
+  void PublishPathPlanningConfigMarkers(const pensa_msgs::PathPlanningConfig &path_planning_config,
+                                        const std::string &inertial_frame_id);
+
+  // Publish markers of the planned trajectory (Rviz visualization)
+  void PublishPathPlanningPathMarkers(const std::vector<Eigen::Vector3d> &path,
+                                      const std::vector<Eigen::Vector3d> &pruned_path,
+                                      const std::string &inertial_frame_id);
+
+  // Calls store_arbiter's service to retrieve path planning config
+  void LoadPathPlanningConfig(const std::string &inertial_frame_id,
+                              pensa_msgs::PathPlanningConfig *path_planning_config,
+                              ros::NodeHandle *nh);
```

Comment:
I am not entirely sure if it is not modified... If I try to declare it as `const ros::NodeHandle &nh`, the compiler complains about using the service client function:
```
error: no matching function for call to ‘ros::NodeHandle::serviceClient(const string&) const’
         nh.serviceClient<pensa_msgs::PathPlanningConfigInFrame>(srv_return_path_planning_config);
```

I am not sure whether the nodehandle is not modified, but it can't be `const`. Without knowing it, I would prefer to keep it as a pointer, which indicates to us that the class might be changed in this function.